core/arch/arm/tee/entry_fast.c

```diff
@@ -127,6 +128,16 @@ static void tee_entry_enable_shm_cache(struct thread_smc_args *args)
 		args->a0 = OPTEE_SMC_RETURN_EBUSY;
 }
 
+static void tee_entry_boot_secondary(struct thread_smc_args *args)
+{
+#ifdef CFG_BOOT_SECONDARY_REQUEST
+	if (generic_boot_core(args->a1, (uint32_t)(args->a3)) == 0)
+		args->a0 = OPTEE_SMC_RETURN_OK;
+	else
+		args->a0 = OPTEE_SMC_RETURN_EBADCMD;
+#endif
```

Comment:
if `CFG_BOOT_SECONDARY_REQUEST` is not enabled, `tee_entry_boot_secondary()` shall load `OPTEE_SMC_RETURN_UNKNOWN_FUNCTION` or not-implemented, if this exists.